src/main/json2xlsx.js

```diff
@@ -21,7 +22,8 @@ const run = async (args) => {
   const builder = new ccdUtils.SpreadsheetBuilder(sourceXlsx);
   await builder.loadAsync();
 
-  const files = fileUtils.listFilesInDirectory(args.sheetsDir, args._);
+  const excluded = args.exclude ? stringUtils.split(args.exclude) : null;
```

Comment:
This looks like boolean flag. It really is list of file exclusions.

Reply:
Renamed to excludedFilenamePatterns